src/Sylius/Bundle/AddressingBundle/composer.json

```diff
@@ -30,8 +30,8 @@
         "stof/doctrine-extensions-bundle": "^1.4",
         "sylius/addressing": "^1.6",
         "sylius/resource-bundle": "^1.7",
-        "symfony/framework-bundle": "^4.4 || ^5.2",
-        "symfony/intl": "^4.4 || ^5.2"
+        "symfony/framework-bundle": "^4.4 || ^5.4",
```

Comment:
I'm not sure it's correct to bump base requirements in minor release, even if that older Symfony version are not supported.

Reply:
in minor, it is ok for me. In patch - not at all

Reply:
So it needs to be reverted as `1.10.4` will be patch release.

Reply:
done 👍